config/env/dev.json

```diff
@@ -56,6 +56,7 @@
     "traceRatio": "@@METRICS_TRACE_RATIO",
     "exportIntervalMillis": "@@METRICS_EXPORT_INTERVAL_MS",
     "exportTimeoutMillis": "@@METRICS_EXPORT_TIMEOUT_MS",
+    "instanceIdentifier": "@@ECS_CONTAINER_METADATA_URI",
```

Comment:
It would be nice to inject this through the Dockerfile via an ENTRYPOINT script and call it `INSTANCE_ID` or something like that.

We do have AWS specific stuff in the env vars and code but parsing a very specific ECS env var directly in the code seems undesirable.

`entrypoint.sh`:
```
#!/bin/bash

uri=$ECS_CONTAINER_METADATA_URI

# Parse the last portion of the URI
value=${uri##*/}

# Remove everything after the hyphen to get the task ID
value=${value%%-*}

export INSTANCE_ID=$value

exec "$@"
```

`Dockerfile`:
```
.
.
.
WORKDIR /

COPY entrypoint.sh .
RUN chmod +x ./entrypoint.sh

COPY runner.sh .

ENTRYPOINT ["./entrypoint.sh"]
CMD [ "./runner.sh" ]
```

Reply:
> We do have AWS specific stuff in the env vars and code but parsing a very specific ECS env var directly in the code seems undesirable.

Oh yeah, good call!